qbittorrent/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: qbittorrent_server_1
+      APP_PORT: 8094
+      PROXY_AUTH_ADD: "false"
+
+  server:
+    image: linuxserver/qbittorrent:4.5.2
```

Comment:
We need to include the multi-arch digest for this Docker image to make sure that everyone installing this app gets the exact same version of the image. To get the digest you can run:
`docker pull linuxserver/qbittorrent:4.5.2`
and the digest will be printed in the output.